prometheus-ksonnet/grafana/dashboards.libsonnet

```
@@ -151,8 +151,10 @@
               path: '/grafana/dashboards-%s' % $.folderID($.mixins[mixinName].grafanaDashboardFolder),
```

If the problem is that the paths on disk need to be unique, and we're already keying mixins by `mixinName` (so that must be unique), then wouldn't it make more sense to use `mixinName` to name the folders so that they're also going to be unique?  AFAIK grafana is fine with multiple providers putting dashboards into the same folder in Grafana, it just doesn't like multiple providers with the same path on disk.  Wouldn't hurt to update line 144 as well: `name: 'dashboards-%s' % $.folderID(mixinName),`
```suggestion
              path: '/grafana/dashboards-%s' % $.folderID(mixinName),
```

---

Thanks, @DanCech. Much better.